### ISSUE TITLE
Specify .coverdata files with a wildcard string

### ISSUE THREAD

**Issue by michalwski** [markusn/coveralls-erl#20]
In my project ([esl/MongooseIM](https://github.com/esl/MongooseIM)) I have 4 different coverdata files. It would be easier for me if I were able to specify all of them by a wildcard (see `filelib:wildcard/1`) . If this is sth you would be interested in, I can submit a PR with this functionality.

**Comment by markusn** [comment]
Sounds good, I will happily merge such a PR! Make sure to retain backwards compatibility with a filename (or list of filenames)

**Comment by fenollp** [comment]
FYI https://github.com/2600hz/kazoo uses this:

```erlang
#!/usr/bin/env escript
%%! +A0 -pa deps/coveralls/ebin
%% -*- coding: utf-8 -*-

-mode('compile').

-export([main/1]).

main(_) ->
    All = filelib:wildcard("*/*/*.coverdata"),
    ok = code:add_pathsa([filename:join(filename:dirname(Path), "ebin") || Path <- All]),
    Id = os:getenv("TRAVIS_JOB_ID"),
    true = is_list(Id),
    coveralls:convert_and_send_file(All, Id, "travis-ci", "").
```

**Comment by michalwski** [comment]
This was implemented in #26
